google/resource_google_project_test.go

```diff
@@ -24,15 +24,42 @@ var (
 	originalPolicy *cloudresourcemanager.Policy
 )
 
+// Test that a Project resource can be created without an organization
+func TestAccGoogleProject_createWithoutOrg(t *testing.T) {
+	if org != "" {
+		t.Skipf("Environment variable GOOGLE_ORG is set")
+	}
+	pid := "terraform-" + acctest.RandString(10)
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			// This step creates a new project
+			resource.TestStep{
+				Config: testAccGoogleProject_createWithoutOrg(pid, pname),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGoogleProjectExists("google_project.acceptance", pid),
+				),
+			},
+		},
+	})
+}
+
```

Comment:
Now that you can create folders, you can add acceptance tests for assigning a project to a folder and moving a project from a folder to an org and vice versa. Since you will be on vacation, I can also take care of writing the new tests. LMK what you prefer.

Reply:
That sounds great. I won't have time for this till middle of next month, and so it would be great if you could add the tests with folder creation and movement between folders. I'll circle back in case you don't have time in the next couple of weeks

Reply:
Oki. I will write these tests today. Thanks!